osm2gtfs/creators/trips_creator.py

```diff
@@ -13,32 +19,301 @@ def __repr__(self):
+                else:
+                    try:
+                        gtfs_trip.AddStopTime(gtfs_stop)
+                    except ValueError:
+                        print("Warning: Could not add a stop to trip.")
+                        print(" " + itinerary_stop.name +
+                              " - " + itinerary_stop.osm_id)
+                        break
+
+                # Add reference to shape
+                gtfs_trip.shape_id = shape_id
+
+                # Add empty attributes to make navitia happy
+                gtfs_trip.block_id = ""
+                gtfs_trip.wheelchair_accessible = ""
+                gtfs_trip.bikes_allowed = ""
+                gtfs_trip.direction_id = ""
+
+            # Calculate all times of stops, which were added with no time
+            Helper.interpolate_stop_times(gtfs_trip)
+        return trips_count
+
+    def _create_gtfs_service_period(self, feed, service):
+        """
+        Generate a transitfeed ServicePeriod object
+        from a time string according to the standard schedule:
+        https://github.com/grote/osm2gtfs/wiki/Schedule
+
+        :return: ServicePeriod object
+        """
+        try:
+            gtfs_service = feed.GetServicePeriod(service)
+            if gtfs_service is not None:
+                return gtfs_service
+        except KeyError:
+            pass
+
+        if service == "Mo-Fr":
+            gtfs_service = ServicePeriod(service)
+            gtfs_service.SetWeekdayService(True)
+            gtfs_service.SetWeekendService(False)
+        elif service == "Mo-Sa":
+            gtfs_service = ServicePeriod(service)
+            gtfs_service.SetWeekdayService(True)
+            gtfs_service.SetWeekendService(False)
+            gtfs_service.SetDayOfWeekHasService(5, True)
+        elif service == "Mo-Su":
+            gtfs_service = ServicePeriod(service)
+            gtfs_service.SetWeekdayService(True)
+            gtfs_service.SetWeekendService(True)
+        elif service == "Sa":
+            gtfs_service = ServicePeriod(service)
+            gtfs_service.SetWeekdayService(False)
+            gtfs_service.SetWeekendService(False)
+            gtfs_service.SetDayOfWeekHasService(5, True)
+        elif service == "Su":
+            gtfs_service = ServicePeriod(service)
+            gtfs_service.SetWeekdayService(False)
+            gtfs_service.SetWeekendService(False)
+            gtfs_service.SetDayOfWeekHasService(6, True)
+        elif service == "Sa-Su":
+            gtfs_service = ServicePeriod(service)
+            gtfs_service.SetWeekdayService(False)
+            gtfs_service.SetWeekendService(True)
+        elif re.search(r'^([0-9]{4})-?(1[0-2]|0[1-9])-?(3[01]|0[1-9]|[12][0-9])$', service):
+            service = service.replace('-', '')
+            gtfs_service = ServicePeriod(service)
+            gtfs_service.SetDateHasService(service)
+        else:
+            raise KeyError("Unknown service keyword: " + service)
```

Comment:
Doesn't this should also handle range of ISO dates like specified in the wiki?

Reply:
@AltNico, Do you think this is a better to be handled in a new issue? Or should it be tread here?

Reply:
I think a new issue would be better.

Reply:
Done #103.